spec/requests/api/v0/articles_spec.rb

```diff
@@ -475,27 +475,27 @@ def post_article(**params)
       end
 
       it "creates an article belonging to the user" do
-        post_article(title: Faker::Book.title)
+        post_article(title: Faker::Book.title, body_markdown: "")
```

Comment:
Since we are adding this as an argument as a default for `post_article` in a few different places, we might consider creating a default params object at the top of these tests with the empty (but valid) `body_markdown`, and then using that inside of the `post_article` method that is defined on line `454`. Alternatively, we could also just set a defaults for `body_markdown` and pass that in as `params: { article: params }`, but also allow it to be overridden.

To be clear, I don't think should block this PR by any means, just figured it would be a nice way to clean this up since we're doing a lot of the same changes in this spec :)